lib/utils.nix

```diff
@@ -125,27 +125,95 @@ rec {
     end
   '';
 
+  /**
+    Convert the given String to a Lua [long literal].
```

Comment:
By all means this is not a requirement, but if you wanted you could mention a potential use-case being to pass a vimscript script to the `vim.cmd()` function? Similar to the note in `toVimscriptHeredoc`'s docstring.

You're welcome to dismiss this feedback without any action.

> For example, you could use this to safely pass a vimscript string to the `vim.cmd` function.

Reply:
I like it!